app/src/main/java/com/jjswigut/matters/database/MatterDataBaseDao.kt

```diff
@@ -0,0 +1,31 @@
+
+
+package com.jjswigut.matters.database
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+
+@Dao
+interface MatterDataBaseDao{
+
+    @Insert
```

Comment:
if we insert a matter / note that already exists, what do we want to happen? might want to specify a conflict resolution strategy